backend/Controllers/SightingController.cs

```diff
@@ -27,20 +37,57 @@ public ActionResult<ListResponse<Sighting>> GetSightingsBySpeciesId([FromRoute]
         {
             var sightings = _sightings.GetSightingsBySpeciesId(speciesId);
             return new ListResponse<Sighting>(sightings);
-        }   
+        }
 
-        [HttpGet("/pending")] 
-        public ActionResult<ListResponse<Sighting>>GetPendingSightings()
+        [HttpGet("/pending")]
+        public ActionResult<ListResponse<Sighting>> GetPendingSightings()
         {
             var pendingSightings = _sightings.GetPendingSightings();
             return new ListResponse<Sighting>(pendingSightings);
-        }   
+        }
 
         [HttpPost]
         public ActionResult CreateSighting([FromBody] CreateSightingRequest createSightingRequest)
-        {   
+        {
             var createdSighting = _sightings.CreateSighting(createSightingRequest);
             return Created("/api", createdSighting);
         }
+
+        [HttpPatch]
+        public ActionResult ConfirmOrRejectSighting([FromHeader] string authorization, [FromBody] ConfirmOrRejectRequest confirmOrRejectRequest)
```

Comment:
~~Good thinking, but remember we want separate PATCH method for a confirmation, and DELETE for a rejection. Therefore, these should both be different methods; one called `ApproveSighting` which is a PATCH request, and one called `RejectSighting` which is a DELETE request.~~

~~Then, we can (and should) get the ID of which sighting we're talking about from the path/route that the request is sent to. For example, sending a DELETE request to `/sightings/3` should reject the sighting with ID 3, and sending a PATCH request to `/sightings/3/approve` should approve the same sighting. (We need to add the `/approve` to the end there, because a PATCH to `/sightings/3` implies an edit of the fields of the sighting, like the image URL or the species, which we're not doing here.)~~

~~Now, we don't need the `ConfirmOrRejectRequest` class _at all_~~

I changed my mind halfway through writing the above. Still read it, as I think it's useful information, but I actually really like your approach with approving/rejecting the sighting all from the same endpoint! So do keep that.

However, the thing about the route still applies! We should get the information about which Sighting we're modifying from the route, i.e. a PATCH request to `/sightings/:id/confirmation` should modify this. You can then retrieve the ID directly as a parameter to the action like this:

```suggestion
        [HttpPatch("/:sightingId/confirmation")]
        public ActionResult ConfirmOrRejectSighting(
          [FromHeader] string authorization,
          [FromRoute] int sightingId,
          [FromBody] ConfirmOrRejectRequest confirmOrRejectRequest
        )
```

and remove it as a field from the `ConfirmOrRejectRequest`.

I also think we should rename the method to `ChangeConfirmationStatus`:
```suggestion
        [HttpPatch]
        public ActionResult ChangeConfirmationStatus([FromHeader] string authorization, [FromBody] ConfirmOrRejectRequest confirmOrRejectRequest)
```